internal/gps/identifier.go

```diff
@@ -167,6 +167,33 @@ type ProjectProperties struct {
 	Constraint Constraint
 }
 
+// GetProjectPropertiesFromVersion takes a Version and returns a proper
+// ProjectProperties with Constraint value based on the provided version.
+func GetProjectPropertiesFromVersion(v Version) ProjectProperties {
```

Comment:
If we're gonna do this, it needs to mirror the `NewSemverConstraint()`, `NewSemverConstraintIC()` duality.

Reply:
Maybe this is only helpful to dep anyway. I'll think about doing as you suggested vs. moving into some other pkg under internal.

Reply:
I ended up moving it back where it was and using a slimmed down version of it in the importers.